csrc/quantization/gptq_marlin/gptq_marlin.cu

```diff
@@ -114,11 +115,20 @@ template <int lut> __device__ inline int lop3(int a, int b, int c) {
   return res;
 }
 
+template <int start_byte, int mask>
```

Comment:
It wouldn't hurt to add a comment on what this does :)

Reply:
@alexm-nm

Reply:
Good idea, added